ml_pipeline_engine/dag/manager.py

```diff
@@ -193,7 +205,7 @@ async def _run_node(
         if self.node_storage.exists_processed_node(node_id):
             logger.debug('Node %s has been executed. Stop new execution', node_id)
 
-            event_lock = self.lock_manager.get_lock(node_id)
+            event_lock = self._lock_manager.evn[node_id]
```

Comment:
evn? You mean the event? Why use the short form?

Reply:
I wanted to decrease the length of the title of the properties. Ok, I'll change it

Reply:
Done